lib/spotlight/dor/indexer.rb

```diff
@@ -77,6 +80,10 @@ def coord_to_decimal point
       dec
     end
     
+    # def add_something_to_solr_doc sdb, solr_doc
```

Comment:
Can you move this example code closer to the `before_index` above? I also wonder if there's a YARD way to say "here's some example code"..